### DISCUSSION TITLE
[Request] : Android 14

### DISCUSSION THREAD

**Discussion by jhay06** [waydroid/waydroid#1424] [General]
I know this is not priority but  would like to request to upgrade to android 14 , android 11 is very old. thank you

**Comment by MuhammadAly11** [comment]
Is there a way to DIY?
Any waydroid alternatives that run the latest version of lingeageOS (21)?

**Reply by Runa-Chin** [reply]
> Is there a way to DIY? Any waydroid alternatives that run the latest version of lingeageOS (21)?

If you're on Windows, use Google Play Games Developer Emulator instead or use AVD (Windows/Linux) if you want to use Android 14.

**Comment by Runa-Chin** [comment]
> I know this is not priority but would like to request to upgrade to android 14 , android 11 is very old. thank you

I guess for now it could only Android 13 and Android 11 could run on Waydroid as [this repo for build the image](https://github.com/waydroid/android_vendor_waydroid) had only Android 10 support, Android 11 support (current OTA production), and Android 13 support (build yourself, and you need more space or buy new SSD if have little capacity).

**Comment by roke-julian-lockhart** [comment]
This would be better filed as [an FR](https://github.com/waydroid/waydroid/issues/new?assignees=&labels=feature&projects=&template=feature_request.yml&title=%5BFeature%5D).

**Comment by supechicken** [comment]
I know this might be a bit late, but here it is :)

https://github.com/supechicken/waydroid-builds/releases/tag/20260118

**Reply by roke-julian-lockhart** [reply]
@supechicken, shall you submit a PR upstream?
